data/hai/hai news.txt

```diff
@@ -165,3 +165,131 @@ news "reformed pirate in Hai space"
+			"Weak human. Why do you dare to come to the land of the true Hai?"
+			"Altered Hai are soft and weak-willed. They care not for true struggle or challenge."
+			"The last time a human called me 'cute', I ran him through with a machete."
+			"I seek challenge, and what greater challenge is there than to oppose the Drak, those with the power of gods?"
+			"Sometimes I miss my family and friends back home, but then I remember that here I am a respected warrior. Back there I was a slave to the Drak."
+			"Our ships can beat anything you stupid monkeys can make."
+			"This sure beats being a factory worker."
+			"Without change, a civilization will decay and fall. This will happen to the fettered Hai, as it has happened to countless empires throughout history."
+			"What are you looking at, monkey?"
+			"I can take on a human like you any day."
+			"You looking for a fight, monkey?"
+		word
+			`"`
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"At first I thought that joining the Unfettered would be fun. Now I can't go back or I'll be killed."
+			"I feel like the only Hai on this planet who regrets joining the Unfettered."
+			"I want to go home."
+		word
+			`"`
+news "unfettered hai after wanderers invasion"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Unfettered warlord"
+			"Hai youth"
+			"Hai warlord"
+	message
+		word
+			`"`
+		word
+			"We shall conquer the worlds to the north as is our right. It is inevitable that the strong shall defeat the weak."
+			"I have fought with the stupid birds to the north. They have no knowledge of proper battle. Our war with them will be swift."
+			"Those overgrown birds have fantastic weaponry, yet they are too cowardly to use it."
+			"I wonder why the birds do not simply fight back. Their weapons are clearly capable of doing so."
+			"I wish I could get my hands on some of that Wanderer tech."
+			"Wanderer worlds are pristine! We have won a great prize in our battle against the stupid birds."
+			"The true Hai will no longer be confined in their Drak-imposed cage."
+			"I'll see great glory in fighting the weak-willed birds."
+			"The worlds up north are nothing like the ones we were confined to. On the Wanderer worlds, the air is clean and the water is as clear as glass."
+		word
+			`"`
+news "unfettered hai after wanderers invasion regret"
+	location
+		government "Hai (Unfettered)"
+	to show
+		has "Wanderers Invaded 0: done"
+	name
+		word
+			"Unfettered fighter"
+			"Unfettered youth"
+			"Hai youth"
+	message
+		word
+			`"`
+		word
+			"I had to shoot down a Wanderer ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
+			"The Wanderers hardly try to fight back. It makes me wonder if they want us to conquer their planets."
```

Comment:
Birds, but also, I'm not sure that first one is entirely true. We haven't heard of any such atrocities being performed against the Wanderers, they seemed to be pretty effective at evacuation.
```suggestion
			"I had to shoot down a bird ship full of civilians when my captain decided to invade their space. I don't know if I can take this anymore."
			"The birds hardly try to fight back. It makes me wonder if they want us to conquer their planets."
```

Reply:
Correct me if I'm wrong, but the player doesn't take part in every evacuation. After all, we're talking planets full of people. I think it's possible that there were some evacuations that didn't go well and were attacked.

Reply:
True, but there aren't a lot of wanderers, I feel like that would get a lot of attention.

Reply:
Are there any solid population figures for the Wanderers?

Reply:
In `mission "Unfettered Jump Drive 3"` the Unfettered say, "They are an old and strong species, but few in number, and those worlds are ours by right."

There are no hard numbers, but this implies that the Wanderers are fewer in number than even the Unfettered.

Reply:
I think that it's plausible that between the Unfettered attacks, Iktat Rek's Molt, and the general chaos of trying to rebuild an army from the ground up in a short timeframe, that something like this happening could just not be mentioned to the player. It doesn't necessarily have to be a large ship, either. I could change it to be some small merchant ship, if that would work.

Reply:
Mmm, at present the wording implies it's an evacuation ship - which is one of the limited number of deep river transports with 166 people on board (or after any one of the rebalances happen, somewhere between 800 and 1400 people, outcome tbd).  

So yeah maybe "one of their merchant ships that was mistaken for a combatant" or something is still like a "oh dear we blew up a bunch of innocent civilians" without being a "oh we actively committed deliberate war crimes against the biggest transport in the galaxy worth of civilians".

Reply:
I edited the line, does it work better now?

Reply:
Oh yes, I like the new version.